runbot_travis2docker/models/runbot_build.py

```diff
@@ -168,6 +169,18 @@ def get_docker_run_cmd(self):
             '-e', 'DB_BACKUP=1',
         ] if not build.is_pull_request and build.repo_id.use_docker_cache \
             else []
+        wl_cmd_env = []
+        if build.branch_id.sync_weblate:
```

Comment:
Use `build.sync_weblate` because we need just create translations from cron job in order to avoid a commit for each build of this branch